src/main/java/jenkins/plugins/logstash/persistence/AbstractLogstashIndexerDao.java

```diff
@@ -57,6 +59,9 @@
   }
 
   @Override
+  @SuppressFBWarnings(
+    value="STCAL_INVOKE_ON_STATIC_DATE_FORMAT_INSTANCE",
+    justification="TODO: not sure how to fix this")
```

Comment:
This is a real defect. SimpleDateFormar is not thread-safe, so this code may fall apart at any moment.
I would recommend just instantiating the object instead of using the static field